specs/1128-EXPT-export_private_key.md

```diff
@@ -0,0 +1,7 @@
+## Export private key
+
+- I can see a notifications warning me about exporting my private keys (<a name="1128-EXPT-001" href="#1128-EXPT-001">1128-EXPT-001</a>)
+- There is a close button on the modal(<a name="1128-EXPT-002" href="#1128-EXPT-002">1128-EXPT-002</a>)
+- I am required to enter my passphrase to export my private keys (<a name="1128-EXPT-003" href="#1128-EXPT-003">1128-EXPT-003</a>)
+- If my passphrase is incorrect I see an error informing me of this (<a name="1128-EXPT-004" href="#1128-EXPT-004">1128-EXPT-004</a>)
+- If I enter the correct passphrase then I see the [reveal hidden information component](./1129-HDCN-hidden_container.md) and a button to close the modal (<a name="1128-EXPT-005" href="#1128-EXPT-005">1128-EXPT-005</a>)
```

Comment:
additional: If my password is blank the export button will remain disabled (and corresponding unit test please)